docs/marketplace.service-management-area.md

```diff
@@ -0,0 +1,73 @@
+---
+id: marketplace.service-management-area
+title: Service Management Area
+---
+
+> Please note that the following functionality is only available to meshStacks that have the **Service Economy** module activated.
+> Reach out to your Customer Success representative if you are interested in using this module.
+
+## Introduction
+
+The Service Management Area can be used by workspaces to offer their [Platform Services](./marketplace.index.md#platform-services) to the rest of the organization.
+For example, the Azure Team in your organization could set up a workspace to offer their Azure [Platform](./administration.platforms.md) from there to
+the rest of the organization. Or the Networking team might want to offer their best-practice Cloud Network [Building Blocks](./administration.building-blocks.md)
+to Application Teams.
+
+Anyone in the organization can offer their valuable services to the rest of the organization through the
+Service Management Area as long as it is approved by someone in the [Admin Area](./administration.index.md).
+
+The Service Management Area can at any time be opened up from the Workspace Control Plane by clicking the
+"Go to Services Management" button as depicted below.
+
+![Introduction](assets/service-management-area/introduction.png)
+
+If you would like to go back again to the regular Workspace view, click on "Go to Workspace Management".
+
+## Types of Platform Services
+
+The following three types of platform services can be created and offered in the Service Management Area:
```

Comment:
We should platform services always in lower case. Can you make that consistent please